src/software/simulation/threaded_simulator_test.cpp

```diff
@@ -146,7 +146,7 @@ TEST_F(ThreadedSimulatorTest, add_robots_and_primitives_while_simulation_running
     threaded_simulator.startSimulation();
     // yield and sleep to give the simulation thread the best chance of running
     std::this_thread::yield();
-    std::this_thread::sleep_for(std::chrono::milliseconds(1500));
+    std::this_thread::sleep_for(std::chrono::milliseconds(2000));
```

Comment:
Why the change in values?

Reply:
makes the test pass more often (it's multithreaded)